### ISSUE TITLE
Tests are broken on BSD because /dev/fd/{N} aren't supported there

### ISSUE THREAD

**Issue by yurivict** [mikefarah/yq#2016]
**Describe the bug**
File descriptors aren't mapped under /dev/fd on BSDs.

Therefore, tests fail with messages like this:
```
yq: error: argument files: can't open '/dev/fd/3': [Errno 2] No such file or directory: '/dev/fd/3'
```

Version of yq: 3.4.1
Operating system: FreeBSD 14.0
Installed via: port

**Input Yaml**
Concise yaml document(s) (as simple as possible to show the bug, please keep it to 10 lines or less)
data1.yml:
```yaml
this: should really work
```

data2.yml:
```yaml
but: it strangely didn't
```

**Command**
The command you ran:
```
yq merge data1.yml data2.yml
```

**Actual behavior**

```yaml
cat: meow
```

**Expected behavior**

```yaml
this: should really work
but: it strangely didn't
```

**Additional context**

**Comment by mikefarah** [comment]
Not sure what would need to be done to fix this - though assuming it's just a development problem, you could run the build in docker, the current Makefile supports this.

**Comment by yurivict** [comment]
No, Docker has nothing to do with this problem, and Docker also doesn't work on FreeBSD.
